src/index.ts

```diff
@@ -57,21 +60,25 @@ async function main() {
     }
     const bundlerConfig: BundlerConfig = {
         ...bundlerConfigSpecResult.value,
-        concurrency,
+        // concurrency,
```

Comment:
@pedrofsantoscom why is concurrency removed here? i thought it was required to speed up the process?

Reply:
doesnt work with the function i'm using. plus concurrency didn't made much difference even on my mac